translation from c00/30/16 to 20 (#66)
IllusionMods/HS2-Translation: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c00/30/16/5/translation.txt

//
// Dumped for Honey Select 2 v1.1.3 by Text Dump v1.4.1
//

> ただただ不愉快です。これ以上、私に関わらないでもらえますか？=I find it simply unpleasant. Could you please not engage me any further?

Reviewer: This reads a bit awkwardly

```suggestion
ただただ不愉快です。これ以上、私に関わらないでもらえますか？=That was unpleasant. Could you please leave me alone from now on?
```

Author: i think it's when the girl is caught but leaves? (maybe it's a talking option, again, i have no idea how this scene plays out). i was thinking she was referring to the whole situation, and thus deciding to go. (fur also has a line where she again says that [h] has left). Maybe "It was unpleasant. Could you please leave me alone from now on?" fits better in this case? or do i keep it as "that"?

Reviewer: This is after you fail to prevent her leaving so `That was unpleasant` seems right as it refers to what just happened. `It was unpleasant` sounds less specific as to what was unpleasant.

Author: yeah, i was going for the less specific route, but i had the wrong premise, as per your explanation, it's very specific indeed :D

フュル=Fur

> //帰られてしまいましたね、残念な事です。=

Reviewer: You commented out the MTL but didn't translate this line

Author: yeah, i already changed that above, it will be propagated by the dupe tool later after approval
